doc/userguide/rules/dcerpc-keywords.rst

```diff
@@ -0,0 +1,67 @@
+DCE/RPC
+=======
+Suricata supports decoding `DCE/RPC <https://en.wikipedia.org/wiki/DCE/RPC>`_ application traffic via the dcerpc application layer protocol. Because of this we are able to detect within DCE/RPC traffic for interface UUIDs, operation numbers, and for bytes within the stub data. This is achieved by using two keywords and a sticky buffer: ``dcerpc.iface``, ``dcerpc.opnum``, and the ``dcerpc.stub_data`` sticky buffer.
```

Comment:
perhaps its good to mention that we support both direct DCERPC (over UDP) and DCERPC over SMB